src/components/Icon/Icon.tsx

```diff
@@ -25,7 +26,7 @@ export type IconSize =
   | 'huge'
   | 'massive'
 
-export interface IconProps extends UIComponentProps {
+export interface IconProps extends UIComponentProps, ColorComponentProps {
```

Comment:
why do we need to introduce this `ColorComponentProps ` type here? It should be enough just to provide `color: true` to `...commonPropTypes.createCommon({`

Reply:
@kuzhelov one is the interface, the other one is the propTypes, makes sense?

Reply:
sorry - yes, absolutely, thanks!